frontend/src/lib/components/LemonCheckbox/LemonCheckbox.scss

```diff
@@ -90,9 +90,11 @@
 
     &.LemonCheckbox--bordered {
         label {
-            padding: 0.75rem;
+            padding: 0.5rem 0.75rem;
             border-radius: var(--radius);
             border: 1px solid var(--border);
+            background: var(--white);
+            line-height: 22px; // 2px less than standard due to border
```

Comment:
IIRC in LemonButton styling we avoid these `height - 2px` numbers by using `height: 1.5rem` and only using horizontal `padding`, so that `box-sizing: border-box` handles this

Reply:
Good call. I was getting confused because of `line-height` vs `height` but with `height` this works much nicer